elastic/kibana
# [SLO] Reset UI for updating outdated SLOs

## Summary

This PR is a follow up to #172224, it adds a UI for resetting the SLO definitions from the previous model. Once #179473 is merged I will rebase this against `main` and convert it from a "draft" PR to "ready to review".

![image](https://github.com/elastic/kibana/assets/41702/daf0591c-272f-40c2-9831-658d7b9b1378)

![image](https://github.com/elastic/kibana/assets/41702/d385396d-d840-4574-942a-b8e51ce66066)

![image](https://github.com/elastic/kibana/assets/41702/729df2a0-61e6-41b3-aaa5-8501e7aa7797)


### Testing

1. Start by loading `main`
2. Ingest some data
3. Create some SLOs
4. Change Kibana from `main` to this PR
5. Visit the SLO page, you should see a banner similar to the screen shot above.
6. Do your best to break this
